pkg/logging/level_log_consumer.go

```diff
@@ -33,8 +33,21 @@ type FilteredLevelLogConsumer struct {
 func (l *FilteredLevelLogConsumer) Log(level LogLevel, message string, fields map[string]interface{}) {
 	if l.level <= level {
 		// prepends the name and log level to the message
-		message = fmt.Sprintf("[%s][%s] %s", level, fields["name"], message)
-		l.logger.Println(message)
+		messBuilder := strings.Builder{}
+		_, _ = messBuilder.WriteString("[")
```

Comment:
"_, _" is not needed here, and errors are not checked, I think fmt.Sprintf will be cleaner here.

Reply:
at the consumer level, there is no telling how many interfaces have been added.  so, a sprintf is not appropriate here.

Reply:
also, the _,_ was a result of go lint.  our linter required it.

Reply:
I think that linters complain about not checking an error. Anyway, WriteString is returning nil error all the time, so there is really no point of checking that error. 

I wanted to simplify this:
```
                 _, _ = messBuilder.WriteString("[")
		_, _ = messBuilder.WriteString(level.String())
		_, _ = messBuilder.WriteString("]")
```
with
``` 
message = fmt.Sprintf("[%d]", level )
_, _ = messBuilder.WriteString(message)
```
maybe it's more readable